src/meta_schedule/database/database.cc

```diff
@@ -25,8 +26,8 @@ namespace meta_schedule {
 
 Workload::Workload(IRModule mod) {
   ObjectPtr<WorkloadNode> n = runtime::make_object<WorkloadNode>();
-  n->shash = tvm::StructuralHash()(mod);
   n->mod = mod;
+  n->shash = ModuleEquality::Create("structural")->Hash(mod);
```

Comment:
Since this constructor is not used anywhere, I'm hard coding the instance here to avoid passing the `mod_eq_name` string. I want to remove this constructor if there is no objection.